.dockerignore

```diff
@@ -0,0 +1,18 @@
+.env*
+.git
+.gitignore
+.codeclimate.yml
+.dockerignore
+.hound.yml
+.travis.yml
+LICENSE.md
+README.md
+docker-compose.*
```

Comment:
`.dockerignore` the next time that we change this file, it won't be efective, right?